src/realm/__tests__/realmReducer-test.js

```diff
@@ -233,5 +235,148 @@ describe('realmReducer', () => {
         check(false, false);
       });
     });
+
+    describe('type `realm`, op `update_dict`', () => {
+      /**
+       * A list of { initialState, eventData, expectedState } with all
+       *   combinations of the given single-property changes.
```

Comment:
I agree, this seems like more complexity than is worth it (given that what's being tested here doesn't really have any tricky interactions.)